src/main/java/net/openhft/chronicle/wire/channel/ChronicleContext.java

```diff
@@ -207,8 +207,7 @@ public ChronicleChannelSupplier newChannelSupplier(ChannelHandler handler) throw
 
         connectionSupplier
                 .protocol(url().getProtocol())
-                .hostname(hostname == null || hostname.isEmpty() ? "localhost" : hostname)
-                .port(port)
+                .addHostnamePort(hostname == null || hostname.isEmpty() ? "localhost" : hostname, port)
```

Comment:
Why? That's not a common use for Builder pattern.

Reply:
It's to add a pair of hostnames/ports, which makes more sense when there are multiple hostnames/ports to be added.